buildspec.json

```diff
@@ -1,30 +1,30 @@
 {
     "dependencies": {
         "obs-studio": {
-            "version": "30.1.2",
+            "version": "30.2.3",
             "baseUrl": "https://github.com/obsproject/obs-studio/archive/refs/tags",
             "label": "OBS sources",
             "hashes": {
                 "macos": "490bae1c392b3b344b0270afd8cb887da4bc50bd92c0c426e96713c1ccb9701a",
-                "windows-x64": "c2dd03fa7fd01fad5beafce8f7156da11f9ed9a588373fd40b44a06f4c03b867"
+                "windows-x64": "a75de0a777013ae5edca526e21b584176bf21606d4d5db23ba4b5a87cfde12e3"
             }
         },
         "prebuilt": {
-            "version": "2024-03-19",
+            "version": "2024-09-05",
```

Comment:
OBS Studio 30.2.3 does not use the 2024-09-05 deps. It uses the 2024-05-08 deps.